kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
+        kordac_extension = KordacExtension()
+        kordac_extension.heading = None
         self.required_files = {}
-        self.html_string = ''
-        html = None
-        ext = KordacExtension()
+
         converter = markdown.Markdown(extensions=[
             'markdown.extensions.fenced_code',
             'markdown.extensions.codehilite',
             'markdown.extensions.sane_lists',
             mdx_math.MathExtension(enable_dollar_delimiter=True),
-            ext])
+            kordac_extension])
+        html = converter.convert(markdown_string)
+
+        result = KordacResult(
```

Comment:
is it typical to format the parameters like this?
Why would you not just write this in one line:
` result = KordacResult(html, kordac_extension.page_heading)`